recipes/alteryx-open-src-update-checker/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "alteryx-open-src-update-checker" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/a/alteryx-open-src-update-checker/alteryx_open_src_update_checker-{{ version }}.tar.gz
```

This should work too!

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

---

Thank you @jaimergp !